src/ai_rest.py

```diff
@@ -28,7 +28,7 @@ def predict_on_images(images):
         "shape": images[0].shape
     })
     request_headers = {"Content-Type": "application/json"}
-    request_url = "https://pepper.lillbonum.se/predict/hand"
+    request_url = "http://host.docker.internal:5000/"
```

Comment:
I think its better to keep the url to external server

Reply:
fixed